package.json

```diff
@@ -58,5 +58,6 @@
       "index.test.js",
       "lib/**/*.test.js"
     ]
-  }
+  },
+  "sideEffects": false
```

Comment:
this must be inside the template package

Reply:
Oh wow I lost attention here, thanks.